frontend/src/metabase/auth/actions.ts

```diff
@@ -13,6 +16,16 @@ import {
 } from "./analytics";
 import { LoginData } from "./types";
 
+export const REFRESH_LOCALE = "metabase/user/REFRESH_LOCALE";
+export const refreshLocale = createThunkAction(
+  REFRESH_LOCALE,
+  () => async (dispatch: any, getState: () => State) => {
+    const userLocale = getState().currentUser?.locale;
+    const siteLocale = Settings.get("site-locale");
```

Comment:
Personal observation: I'm so used to seeing `MetabaseSettings`, so `Settings` made me confused for a moment. Should we use `MetabaseSettings` here too for consistency?

Reply:
Sure, fixed